.env.example

```diff
@@ -1,3 +1,3 @@
-NEXT_PUBLIC_AWS_API_URL=
-NEXT_PUBLIC_APP_DASHBOARD_DOMAIN=
-NEXT_PUBLIC_APP_DOMAIN=
\ No newline at end of file
+NEXT_PUBLIC_AWS_API_URL= http://localhost:8000/local
```

Comment:
dont change the `.env.example` file. Keep it the same as below

```
EXT_PUBLIC_AWS_API_URL=
NEXT_PUBLIC_APP_DASHBOARD_DOMAIN=
NEXT_PUBLIC_APP_DOMAIN=
```

Reply:
Okay